mitodl/ocw-zendesk-theme
# Sync the ocw zendesk theme with upstream.

### What are the relevant tickets?

[#3841](https://github.com/mitodl/hq/issues/3841) 


### Description (What does it do?)
Here is the Testing URL of re-synced theme: https://mitodl5553.zendesk.com/hc/en-us

This PR syncs the ocw Zendesk theme with the upstream. It maintains the existing design sync and incorporates updates and features from the Zendesk upstream theme. To see a list of all the updates and features, please refer to the changelog [here](https://github.com/zendesk/copenhagen_theme/blob/master/CHANGELOG.md).

### How should this be manually tested?

1. Homepage Changes:

- Visit the [old ocw Zendesk home page](https://mitocw.zendesk.com/hc/en-us) and compare it with the [new ocw Zendesk home page](https://mitodl5553.zendesk.com/hc/en-us).
- Ensure that the clear search feature is functional on multiple pages.

2. Submit a Request Page:

- Navigate to the [old submit a request page](https://mitocw.zendesk.com/hc/en-us/requests/new) and compare it with the [new submit a request page](https://mitodl5553.zendesk.com/hc/en-us/requests/new).

- Ensure that the layout and functionality of the new submit a request page match the expected changes and incorporate new changes if any.

3. "Sign In" link:
- Confirm that the "Sign In" link is not visible in the test environment and will not be present when deployed to production or verify that the "Sign In" link is disabled in the header for both desktop and mobile views. 

Overall Testing:

- Conduct general testing of the Zendesk theme to ensure that all existing features are functioning correctly after the sync with the upstream.
- Test any other specific features or functionalities mentioned in the changelog or observed during the sync process.

### Screenshots (if appropriate):

**Old ocw zendesk home page:**
![image](https://github.com/mitodl/ocw-zendesk-theme/assets/41048311/e13c0921-e5bd-47e7-af0a-08d33f13e3dc)

**New ocw zendesk home page:**
![image](https://github.com/mitodl/ocw-zendesk-theme/assets/41048311/43b99f7a-9e57-4807-bd32-e38090354612)

**Old submit a request page:**
![image](https://github.com/mitodl/ocw-zendesk-theme/assets/41048311/ea09596f-e24d-4c99-8162-66697b44bc2d)

**New submit a request page:**
![image](https://github.com/mitodl/ocw-zendesk-theme/assets/41048311/3e9581b5-afca-463b-9960-d46942d76f83)

**Old ocw zendesk categories page:**
![image](https://github.com/mitodl/ocw-zendesk-theme/assets/41048311/73a3eb96-3020-4a1a-a95a-46076862747d)

**New ocw zendesk categories page:**
![image](https://github.com/mitodl/ocw-zendesk-theme/assets/41048311/2169b954-b98f-483c-8073-aa40013779df)

**Old mobile view:**
![image](https://github.com/mitodl/ocw-zendesk-theme/assets/41048311/d59dd7f5-d094-4097-9f5e-09e1af54faa9)

**New mobile view:**
![image](https://github.com/mitodl/ocw-zendesk-theme/assets/41048311/c4fa0eca-7eb6-4121-8c15-585e46ad11e6)
